src/Avalonia.Controls.DataGrid/DataGrid.cs

```diff
@@ -5674,6 +5680,35 @@ private void VerticalScrollBar_Scroll(object sender, ScrollEventArgs e)
             VerticalScroll?.Invoke(sender, e);
         }
 
+        //TODO: Ensure right button is checked for
```

Comment:
Does this still need to be implemented or is the TODO out of date?

Reply:
You must ask the person who added the following (see below). I just copied it for consistency. I have no idea what it means but when it exists for left click then I've added it for right click as well assuming it means something.
https://github.com/AvaloniaUI/Avalonia/blob/a85654e259973e938c398b5fca56c35f393a2196/src/Avalonia.Controls.DataGrid/DataGrid.cs#L3042